app/views/events/edit.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>This is the edit page</h1>
+<div class="container w-50">
+  <%= simple_form_for @event do |f| %>
+    <%= f.association :boardgame, :label => "Name of Board game", :include_blank => false %>
```

Comment:
`<%= f.association :boardgame, label: "Name of Board game", include_blank: false%>`